.github/workflows/ci.yml

```diff
@@ -8,13 +8,14 @@ on:
 
 jobs:
```

Comment:
Should we also add a check for compiling to WASM target, similar to how we do in the  [miden repo](https://github.com/maticnetwork/miden/blob/main/.github/workflows/ci.yml#L76)?

Reply:
Target added!